test/integration/react-streaming-and-server-components/test/runtime.js

```diff
@@ -9,4 +13,31 @@ export default async function runtime(context, { runtime }) {
       expect(html2).toContain('Runtime: Node.js')
     })
   }
+  if (runtime === 'edge' && env === 'prod') {
```

Comment:
Should nodejs runtime be included here as well?

Reply:
This test is mainly for mixed runtimes. Since the page runtime of /runtime and /runtime-rsc is `nodejs`, we want to test the case that all other pages are using the `edge` runtime. If we set the global runtime to `nodejs`, it's not affected by this PR and already works today.